src/equations/ideal_glm_mhd_2d.jl

```diff
@@ -489,6 +678,40 @@ phi^L 1/2 (B^L + B^R)_{normal} - phi^L B^L+{normal} = phi^L 1/2 (B^R - B^L)_{nor
 end
 
 
+# Compute surface nonconservative "flux" computation in the normal direction (2D version)
+# Note, due to the non-uniqueness of this term we cannot use any fancy rotation tricks.
+# Also, this currently only performs the mode = :weak version
```

Comment:
Maybe add an appropriate assertion in this case?

Reply:
Done